Translation:Chinese.md

```diff
@@ -446,18 +445,17 @@ Hello Latex, This is my second part.
 
 #### 版本(Distributions)
 
+* [TeX Live](https://www.tug.org/texlive/) 适用于所有平台（包括 Windows、Linux 和基于 Unix 的系统）。
 * [MiKTeX](https://miktex.org/about) 适用于 Windows。
-* [TeX Live](https://www.tug.org/texlive/) 适用于 Linux 和基于 Unix 的系统。
 * [MacTeX](https://tug.org/mactex/) 适用于 macOS。
-* [ShareLaTeX](https://www.sharelatex.com/) — 一个在线编辑器。
```

Comment:
@inkydragon Why have you removed ShareLaTeX link and moved TeX Live up the list? It is not the best idea to mix improvements of the translation with changes in the content. If you'd like to propose changes to content, do it for the original English text.
@luong-komorebi FYI

Reply:
Just open the link, and I've writen the reason in the commit msg "> ShareLaTeX is now part of Overleaf". 

About "moved TeX Live up", originally I would like to add a link that discussed why we should use TeX Live rather than other distributions, but i don't find a good place to add this link, so I just change the order. 

Maybe the best idea is fork this repo and do those localize change. (~~or people who use github already have the ability to choose a distributions by themself, and can deal with the problems they 
 may encounter~~)

Or maybe I should revert those two changes rather than argue.

Reply:
ShareLaTeX is part of Overleaf now so this is fine. For the order, I originally don't place any specific order on it. The author putting Tex Live on top of the list for me would be fine but of course the original order should be kept.

Reply:
Thank you for noticing this detail, @i-ky